src/components/views/auth/VectorAuthFooter.js

```diff
@@ -27,11 +29,27 @@ module.exports = React.createClass({
     },
 
     render: function() {
+        const brandingConfig = SdkConfig.get().branding;
+        let links = [
+            {"text": "blog", "url": "https://medium.com/@RiotChat"},
+            {"text": "twitter", "url": "https://twitter.com/@RiotChat"},
+            {"text": "github", "url": "https://github.com/vector-im/riot-web"},
+        ];
+
+        if (brandingConfig && brandingConfig.welcomeLinks) {
```

Comment:
I think `authFooterLinks` would be a more precise name for the option, since the welcome page is only shown when first loading the app, but these links are on screen throughout any auth flows.

Reply:
done :)